packages/react/src/components/builder-block.component.tsx

```diff
@@ -181,7 +181,15 @@ export class BuilderBlock extends React.Component<
     const reversedNames = sizeNames.slice().reverse();
     const styles: any = {};
     if (responsiveStyles) {
+      const contentHasXSmallBreakpoint = Boolean(
```

Comment:
Suggestion: we could avoid this flag and the check in the for loop later. That way we will treat xsmall as existing medium or small sizes.

Reply:
But in the loop also we'll need to have a separate if clause `size === 'xsmall'` right ? Currently it is treating the xsmall the same way as medium / small sizes, but with a early return in the case when content doesn't have breakpoints xsmall.

Reply:
Can we add a Note/comment saying that this check can be removed once we make `xsmall` becomes GA

Reply:
done